### PR TITLE
Enhancement/update into slide

### PR DESCRIPTION
**Pull request by munroTom**
Styles are generally more responsive as well
Screenshot 2020-06-13 at 14 53 21
Screenshot 2020-06-13 at 14 53 29
Screenshot 2020-06-13 at 14 53 38

### CODE REVIEW

**Comment by lizheard1** [comment]
@munroTom great! What do you mean by responsive styles? As in they fit to different screen sizes better? A few small comments:

- 1st slide: last sentence should be 'map' not 'maps'
- 2nd slide: can wording be moved down slightly so a bit more of a gap between 'Planet Patrol' and the wording? 
- Slide 3 - is this orange the brand guidelines orange? - please could the colour be matched to the orange on the 'snap it'

@joe-mcguinness do you have any feedback from a stylistic perspective?

File: src/components/common/Logo/Logo.tsx [hunk 1]
```diff
@@ -1,19 +1,225 @@
...
+};
+type Keys = keyof typeof COLORS;
+type Fill = typeof COLORS[Keys];
+type Props = { fill?: Fill; className?: string };
+
+export default function Logo({ fill = COLORS.TEAL, className }: Props) {
   const styles = useStyles();
-  return <img className={styles.logo} src={logo} alt={"Planet Patrol"} />;
+  return (
+    <svg
```
Comment (neilfulwiler) [inline src/components/common/Logo/Logo.tsx:25]: can we put this in a separate .svg file?

**Comment by joe-mcguinness** [comment]
> @munroTom great! What do you mean by responsive styles? As in they fit to different screen sizes better? A few small comments:
> 
> * 1st slide: last sentence should be 'map' not 'maps'
> * 2nd slide: can wording be moved down slightly so a bit more of a gap between 'Planet Patrol' and the wording?
> * Slide 3 - is this orange the brand guidelines orange? - please could the colour be matched to the orange on the 'snap it'
> 
> @joe-mcguinness do you have any feedback from a stylistic perspective?

Completely agree with your points, plus we should really be using the Brown STD typeface here. I've attached the brand guidelines here for reference.
[Plastic Patrol Brand Guidelines  (2) (1).pdf](https://github.com/PlasticPatrol/PlasticPatrolWeb/files/4813357/Plastic.Patrol.Brand.Guidelines.2.1.pdf)

**Comment by munroTom** [comment]
@joe-mcguinness do you have a copy of the typeface asset that can be used in the app? I can't find it online / the figma project you sent

File: src/components/common/Logo/Logo.tsx (hunk 1 above)
Comment (munroTom) [reply src/components/common/Logo/Logo.tsx:25]: I tried doing that but then it wasn't super easy to override the fill colour, so in line 30 we set the fill dynamically from the `COLOR` options further up, I'm not super familiar with using svgs inline so there may be a nicer way to do it that I'm not aware of!

**Comment by munroTom** [comment]
>What do you mean by responsive styles? As in they fit to different screen sizes better?

Yep that's it! Hopefully these should look ok on any screen size, whereas before the spacing/ sizes of assets didn't really change leading to a lot more wasted space on larger screens

**Comment by joe-mcguinness** [comment]
[Brown STD.zip](https://github.com/PlasticPatrol/PlasticPatrolWeb/files/4842039/Brown.STD.zip)
@munroTom  attached the font files

**Comment by joe-mcguinness** [comment]
> @munroTom great! What do you mean by responsive styles? As in they fit to different screen sizes better? A few small comments:
> 
> * 1st slide: last sentence should be 'map' not 'maps'
> * 2nd slide: can wording be moved down slightly so a bit more of a gap between 'Planet Patrol' and the wording?
> * Slide 3 - is this orange the brand guidelines orange? - please could the colour be matched to the orange on the 'snap it'
> 
> @joe-mcguinness do you have any feedback from a stylistic perspective?
I think the spacing on the 2nd and 3rd screenshots could be improved, and the colour from guidelines is here:
Screenshot 2020-06-28 at 10 59 25

**Comment by munroTom** [comment]
Sorry I didn't update the screenshot - I've updated the orange though, will add the typeface in when I get a moment and see what I can do with the spacing, I've added some more margin between the logo and first bit of text on each slide

**Pull request merged**